epilepsy12/common_view_functions/aggregate_by.py

```diff
@@ -907,7 +907,7 @@ def create_KPI_aggregation_dataframe(
         )
 
     if model_aggregation_2:
-        if constants_list2 == TRUSTS:
```

Comment:
This check was redundant - we only accessed `constants_list2` if this was true.

I think it would be clearer to remove the checks against `constants_list1` too but that's for a future PR